### PR TITLE
Update Analytics plugin for Accelerate

### PR DESCRIPTION
**Pull request by roborourke**
Adds handling for accelerate tier and above only plus docs and config option for the dashboard override.

Fixes https://github.com/humanmade/product-dev/issues/854

### CODE REVIEW

File: inc/native/namespace.php [hunk 1]
```diff
@@ -17,8 +17,16 @@
  * Setup integration.
  */
 function bootstrap() {
+	// Only load native analytics solution for Accelerate or higher.
+	if ( Altis\get_feature_tier() === 'plus' ) {
```
Comment (rmccue) [inline inc/native/namespace.php:21]: Seeing as we don't currently have this data set up, can we use analytics.enabled here instead? Given this is in the analytics module, we can control the tiering at the higher level

Comment (roborourke) [reply inc/native/namespace.php:21]: That's essentially what's required to get to this function anyway.

Comment (roborourke) [reply inc/native/namespace.php:21]: So how can I tell what's happening at a higher level if not via the tier constants?

Comment (rmccue) [reply inc/native/namespace.php:21]: Yeah, so would make sense to just drop the check I think then?

**Review by rmccue — APPROVED** [review]

**Pull request merged**